Realm/RLMUtil.hpp

```diff
@@ -100,9 +108,14 @@ static inline NSString *RLMTypeToString(RLMPropertyType type) {
 static inline NSString * RLMStringDataToNSString(realm::StringData stringData) {
     static_assert(sizeof(NSUInteger) >= sizeof(size_t),
                   "Need runtime overflow check for size_t to NSUInteger conversion");
-    return [[NSString alloc] initWithBytes:stringData.data()
-                                    length:stringData.size()
-                                  encoding:NSUTF8StringEncoding];
+    if (stringData.is_null()) {
```

Comment:
how did this even pass CI given that build.sh doesn't specify a null-enabled version of core?